pkg/controllers/nodeclaim/disruption/emptiness.go

```diff
@@ -46,6 +46,7 @@ type Emptiness struct {
 //nolint:gocyclo
 func (e *Emptiness) Reconcile(ctx context.Context, nodePool *v1beta1.NodePool, nodeClaim *v1beta1.NodeClaim) (reconcile.Result, error) {
 	hasEmptyCondition := nodeClaim.StatusConditions().GetCondition(v1beta1.Empty) != nil
+	reconcileLog := log.FromContext(ctx).V(1)
```

Comment:
Same comment here. Do we need to pull the logger out of the context and then just make everything debug, or can we just call this imperatively where we need it?